_includes/releases/v23.1/v23.1.0-alpha.8.md

```diff
@@ -0,0 +1,139 @@
+- Some KV server [cluster settings](../v23.1/cluster-settings.html)  are now system only. These settings could previously be written or read by tenants, however writing to these settings have no effect. [#98353][#98353]
+- Checkpoint directories that can be created in the rare event of range inconsistency are now clearly indicated as `pending` until they are fully populated. This helps operators  distinguish valid checkpoints from corrupted ones. [#99119][#99119]
+- [Prometheus](../v23.1/monitor-cockroachdb-with-prometheus.html) metrics available at the `_status/vars` path now contain a `node_id` label that identifies the node they were scraped from. [#99235][#99235]
+
+<h3 id="v23-1-0-alpha-8-db-console-changes">DB Console changes</h3>
+
+- New data is not auto fetched every 5m on the [Statement & Transaction Fingerprints](../v23.1/ui-statements-page.html#statement-fingerprint-page) pages. [#98331][#98331]
+- Updated the column selector icon to show the gear icon and "Columns". [#98826][#98826]
+- Updated the Jobs table column name from "Last Modified Time" to "Last Execution Time". [#99021][#99021]
+- On the [SQL Activity Fingerprints](../v23.1/ui-overview.html#sql-activity) pages, users will not see stats that have not yet been flushed to disk. [#98815][#98815]
+- Users can now request `top-k statements by % runtime` on the SQL Activity Fingerprints pages. [#98815][#98815]
+- Added Search Criteria to the [Statements](../v23.1/ui-statements-page.html) and [Transactions](../v23.1/ui-transactions-page.html) pages, and updated the UX with improvements. [#98815][#98815]
+- Timeseries metrics in [DB Concole](../v23.1/ui-overview.html) will now show tenant-specific metrics. For the system tenant, these timeseries will be aggregated across all tenants. For a secondary tenant, only metrics belonging to that particular tenant will be shown. [#98077][#98077]
+- Added badges for each selected filter on the [SQL Activity](../v23.1/ui-overview.html#sql-activity) and [Insights](../v23.1/ui-insights-page.html) pages. [#98988][#98988]
+- Data on the [Statement Fingerprint Details](../v23.1/ui-statements-page.html#statement-fingerprints-view) page will now not update automatically every 5mins. [#99298][#99298]
+- The default request sort for the [Statement Fingerprints Overview](../v23.1/ui-statements-page.html#statement-fingerprints-view) page is now `% of All Runtime`. [#99298][#99298]
+- Fixed a bug where the table's `CREATE` statement would not display correctly on the [Table Details](../v23.1/ui-databases-page.html#table-details) page. [#99434][#99434]
+- Added an assertion on the KV side to prevent other existing or future attempts of LeafTxn issuing locking requests. This ensures the KV API is used as agreed upon and can be helpful in debugging latency issues caused by holding locks. [#99412][#99412] 
+- Increased the TestParsingErrorHandling metric to `500ms` to allow the file to re-read before checking the value of the error metrics. [#99050][#99050] 
```

Comment:
Let's drop this. See https://github.com/cockroachdb/cockroach/pull/99050#issuecomment-1490638225